.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -1,5 +1,6 @@
 name: 🐞 Bug Report
 description: Report a bug if something isn't working as expected in WooCommerce Core.
+title: "[BUG]: "
```

Comment:
Apologies @ralucaStan, on second thoughts maybe it is wrong to have it uppercase as this implies it is an acronym. As you noted in your PR description the enhancement title isn't in upper case. We should ignore my suggestion of matching the DOCS BUG and stick with your original `Bug`. Sorry for the slow turnaround on this review :)

Reply:
I definitely missed your comment, so I am just coming to this. Once you can 👍 I can merge this small PR. Thank you again Tam